eng/pipelines/runtime-linker-tests.yml

```diff
@@ -75,7 +75,7 @@ jobs:
       testGroup: innerloop
       timeoutInMinutes: 120
       nameSuffix: Runtime_Release
-      buildArgs: -s mono+libs -c $(_BuildConfig) -p:WasmBuildNative=false
+      buildArgs: -s mono+libs -c $(_BuildConfig) -p:WasmBuildNative=false -p:UsingBrowserRuntimeWorkload=false
```

Comment:
@lewing @radical - do we still need this change to the .yml file with all the other change to the MSBuild targets/props below?